test/sai_test/config/route_configer.py

```diff
@@ -38,10 +38,17 @@ def t0_route_config_helper(test_obj, is_create_route=True, is_create_route_for_l
 
         ip_addr2 = '10.1.2.100'
         mac_addr2 = '02:04:02:01:02:01'
-        route_configer.create_route_and_neighbor_entry_for_port(ip_addr=ip_addr1, 
+        route_configer.create_route_and_neighbor_entry_for_port(ip_addr=ip_addr2, 
             mac_addr=mac_addr2, 
             port_id=test_obj.lag2.lag_id, 
             virtual_router_id=test_obj.default_vrf)
+        
+        port21_addr = '192.168.0.1'
```

Comment:
for those common config items, we should maintain them in a list that can be operated in the different test step, like testMethod and teardown